packages/story-editor/src/components/form/media.js

```diff
@@ -76,6 +77,7 @@ const MediaInputField = ({
       ref={forwardedRef}
       value={value === MULTIPLE_VALUE ? null : value}
       isRTL={isRTL}
+      leftOffset={leftOffset}
```

Comment:
Media Input uses the tooltip from design system because it gets used in both the dashboard and the editor, so we need to manually pass through `isRTL` and `leftOffset`